### DISCUSSION TITLE
How to Close Web Browser

### DISCUSSION THREAD

**Discussion by Ragnoraq** [rern/rAudio#787] [Tips]
I have an RPi-3 connected to the official 7" display running rAudio-1. I inadvertently opened a web browser to last.fm and cannot figure out how to close the browser. How do you close the web browser?

**Comment by rern** [comment]
Features > Browser on RPi > disable then enable again
Or just reboot.

By the way, the last.fm link will be disabled on local display by next update.

**Reply by fg-ok** [reply]
There is the same problem with the link to this discussion board at the logo in the top left (#logo) as well.
Using a local display, tapping on the logo by accident leads to a new browser window with no possibility to get back.

Maybe it's the same solution to fix this?
